### ISSUE TITLE
Use responsive capabilities of the Grid system

### ISSUE THREAD

**Issue by Ladonasc** [braincrafted/bootstrap-bundle#386]
Hi,

first of all thank you for the work you did on this bundle, which save a lot of times.

My question concern the `label_col` and `widget_col` options available for several components (form_row, form_label, form_widget etc...).

Using them influence the CSS class applied on the element : `col-(xs|sm|md|ls)-[1-12]`.

Unless I miss something it could be only _one_ class applied. Since then we cannot use several col-##-## class to specify several rendering layout depending the device :

```html
<div class="form-group">
    <label class="control-label col-sm-4 col-md-3">Firstname :</label>
    <div class="col-sm-6 col-md-7">
        <input type="text" name="firstname" />
    </div>
</div>
```

Is there an option I didn't saw ?

**Comment by vrnagy** [comment]
You can use multiple classes on the elements. It's used for Mobile and Desktop views.
Example: http://getbootstrap.com/css/#grid-example-mixed

**Comment by Ladonasc** [comment]
Yup it's what I point out on the first message, it seems that current implementation does not allow to specify multiple class to be applied on the element :(
